keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+/* Copyright 2018 GoTakigawa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #include QMK_KEYBOARD_H
+ #include "eeconfig.h"
+
+ extern keymap_config_t keymap_config;
+
+ // Each layer gets a name for readability, which is then used in the keymap matrix below.
+ // The underscores don't mean anything - you can have a layer called STUFF or any other name.
+ // Layer names don't all need to be of the same length, obviously, and you can also skip them
+ // entirely and just use numbers.
+ #define _QWERTY 0
+ #define _COLEMAK 1
+ #define _DVORAK 2
+ #define _LOWER 3
+ #define _RAISE 4
+ #define _EUCALYN 5
+ #define _ADJUST 16
+
+ enum custom_keycodes {
+   QWERTY = SAFE_RANGE,
+   COLEMAK,
+   DVORAK,
+   LOWER,
+   RAISE,
+   EUCALYN,
+   ADJUST,
+ };
+
+ // Fillers to make layering more clear
+ #define _______ KC_TRNS
+ #define XXXXXXX KC_NO
+ #define LOWER MO(_LOWER)
+ #define RAISE MO(_RAISE)
+
+ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+ /* Qwerty
+  * ,---------------------------------------------------------------------.
+  * |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   -  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_QWERTY] = LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_QWERTY] = LAYOUTortho_4x10( \
```